This patch changes the background color of the video output to black to match the output of the FPGA synthesized video component.

diff --git a/src/main/java/com/cburch/logisim/std/io/Video.java b/src/main/java/com/cburch/logisim/std/io/Video.java
index 7824a942a..a9e6e76f5 100644
--- a/src/main/java/com/cburch/logisim/std/io/Video.java
+++ b/src/main/java/com/cburch/logisim/std/io/Video.java
@@ -417,7 +417,7 @@ class Video extends ManagedComponent implements ToolTipMaker, AttributeListener
 
     public void reset() {
       final var g = img.getGraphics();
-      g.setColor(Color.YELLOW);
+      g.setColor(Color.BLACK);
       g.fillRect(0, 0, img.getWidth(), img.getHeight());
     }
 
